src/screens/Rari/RariInfo.js

```diff
@@ -101,17 +97,13 @@ const RariLogo = styled(CachedImage)`
 `;
 
 const RariInfoScreen = ({
-  fetchRariFundBalance,
-  isFetchingFundBalance,
-  rariFundBalance,
   baseFiatCurrency,
   rates,
+  rariFundBalance,
 }: Props) => {
-  useEffect(() => {
-    fetchRariFundBalance();
-  }, []);
-
   const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
+  const totalRariFundBalance = (Object.values(rariFundBalance): any).reduce((sum, balance) => sum + balance, 0);
+  const totalRariFundBalanceInFiat = convertUSDToFiat(totalRariFundBalance, rates, fiatCurrency);
```

Comment:
Just curious to know what happens if this fails? Do we have a fallback value or logic?

Reply:
`rariFundBalance` is kept in redux store and persisted, so if fetching fund balance fails (or any other rari data, like user's balances, prices etc.), 0 will be showed or older value that was fetched recently.
Additionally every time it fails a toast is displayed like `Cannot fetch Rari data. Please try again` or graph error box, if graph call failed.